### PR TITLE
Design protobufs to add, move, remove robots and move ball

### PR DESCRIPTION
**Pull request by HubertTheodore**
### Description

Added protobuf for adding, moving, and removing robots and ball movement.

### Resolved Issues

resolves #2484 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x ] **Remove all commented out code**
- [x ] **Remove extra print statements**: for example, those just used for testing
- [x ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/proto/move_robot_and_ball_messages.proto [hunk 1]
```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package TbotsProto;
+
+import "proto/geometry.proto";
+
+message MoveRobotMessage
```
Comment (akhilveeraghanta) [inline src/proto/move_robot_and_ball_messages.proto]: you can remove the Message suffix, `MoveRobot` should be enough

File: src/proto/move_robot_and_ball_messages.proto [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{
```
Comment (akhilveeraghanta) [inline src/proto/move_robot_and_ball_messages.proto:14]: we need a field here for robot id (to identify which robot to move)

File: src/proto/move_robot_and_ball_messages.proto [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    Point global_position                   = 1;
+    Angle global_orientation                = 2;
+    Vector global_velocity                  = 3;
+    AngularVelocity global_angular_velocity = 4;
```
Comment (akhilveeraghanta) [inline src/proto/move_robot_and_ball_messages.proto:18]: lets add another field here for Team color, looks like that's also needed to figured out which team to control

```
enum TeamColour                                                                                                                                                                                           
{                                                                                                                                                                                                                                                                                                                                                                          
    YELLOW = 1;                                                                                                                                                                                     
    BLUE = 2;                                                                                                                                                                                       
}   
```

And then add a field

`TeamColour team_colour = 5`;

**Review by akhilveeraghanta — CHANGES_REQUESTED** [review]
looks great! just a couple of comments on MoveRobot

File: src/proto/move_robot_and_ball_messages.proto [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+package TbotsProto;
+
+import "proto/geometry.proto";
+
+message RobotId
+{
+    uint32 robot_id = 1;
+}
+
```
Comment (jonathanlew) [inline src/proto/move_robot_and_ball_messages.proto]: this doesn't seem to be used anywhere?

**Review by akhilveeraghanta — DISMISSED** [review]
lgtm!

File: src/proto/move_robot_and_ball_messages.proto [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+message MoveRobot
+{
+    Point global_position                   = 1;
+    Angle global_orientation                = 2;
+    Vector global_velocity                  = 3;
+    AngularVelocity global_angular_velocity = 4;
+    TeamColour team_colour                  = 5;
+    RobotId robot_id                        = 6;
```
Comment (jonathanlew) [inline src/proto/move_robot_and_ball_messages.proto]: not sure why this can't be
```suggestion
    uint32 robot_id                        = 6;
```

**Review by akhilveeraghanta — DISMISSED** [review]

File: src/proto/move_robot_and_ball_messages.proto (hunk 4 above)
Comment (jonathanlew) [reply src/proto/move_robot_and_ball_messages.proto]: bumping

**Review by jonathanlew — COMMENTED** [review]
one more comment here

**Review by akhilveeraghanta — APPROVED** [review]

**Review by jonathanlew — APPROVED** [review]

**Pull request merged**